templates/404.html

```diff
@@ -1,23 +1,21 @@
 {% extends 'base.html' %}
 
 {% block content %}
-    <section class="error">
-            <div class="error__box-img">
-                <div class="container">                                   
-                    <h1 class="error__heading">Oops!<br> 404</h1>
-                    <h3 class="error__text">
-                        It looks like you got lost. <br>
-                        No worries! Just click the button to get back.
-                    </h3>
-                    <!-- Href Home Button -->
-                    <div> 
-                        <button class="button">
-                            <a href="index.html" aria-label="Get back!">
-                                Get back!
-                            </a>
-                        </button>
-                    </div>
-                </div>                
+    <section class="error-notfound">            
+        <div class="error__container error__box-img">                                   
+            <h1 class="error__heading">Oops!<br> 404</h1>
+            <h3 class="error__text">
+                It looks like you got lost. <br>
+                No worries! Just click the button to get back.
+            </h3>
+            <!-- Href Home Button -->
+            <div> 
+                <button class="error__button error__button--notfound">
+                    <a href="index.html" aria-label="Get back!">
```

Comment:
Can you change this href attribute values to be:
```href="{{url_for('index')}}"``` 

This is the only way a href will work with Flask 😁